app/src/androidTest/java/ch/epfl/sdp/blindly/BlindlyTestRunner.kt

```diff
@@ -5,9 +5,20 @@ import android.content.Context
 import androidx.test.runner.AndroidJUnitRunner
 import dagger.hilt.android.testing.HiltTestApplication
 
-// A custom runner to set up the instrumented application class for tests.
+/**
+ * The test runner for Blindly
+ */
 class BlindlyTestRunner : AndroidJUnitRunner() {
 
+    /**
+     * Create a hilt application to be used for the tests
+     *
+     * @param ClassLoader the class loader for the app
```

Comment:
I believe you're supposed to put the name of the argument rather than the class of the argument after the @param
(at least it's this way with the Kdoc plugin...)

Reply:
My bad, I should have taken more time and be more careful while doing it.